source/math/FloatRandomGenerator.ooc

```diff
@@ -50,14 +58,16 @@ FloatGaussianRandomGenerator: class extends FloatRandomGenerator {
 	_backend: FloatUniformRandomGenerator
 	_mu := 0.0f
 	_sigma := 1.0f
+	_secondValue : Float
+	_hasSecond := false
```

Comment:
The Box-Muller transform takes two uniformly distributed numbers and generates two normally distributed numbers: https://en.wikipedia.org/wiki/Box%E2%80%93Muller_transform
We only return one of them, so we save the second value for the next call.